response_operations_ui/views/social/view_social_case_model.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from structlog import wrap_logger
+
+from response_operations_ui.common.mappers import map_social_case_status
+from response_operations_ui.controllers import case_controller, sample_controllers
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def build_view_social_case_model(case_id):
```

Comment:
I think the generally accepted term for what you're building here (to pass into a template) is "context" rather than model.